question-6/js/script.js

```diff
@@ -10,25 +10,25 @@ addTwoNumbers();
 
 /*a function that takes a number as a parameter and 
 returns the square root of that number*/
+
 let squareNum = isSquare(4)
```

Comment:
The variable name says `isSquare()` but the function returns the **square root** of a number. 
You should consider renaming the variable to something related to *square root* not *square*.